pkgs/desktops/xfce/thunar-plugins/media-tags/default.nix

```diff
@@ -0,0 +1,40 @@
+{ lib
+, mkXfceDerivation
+, gtk3
+, thunar
+, exo
+, libxfce4util
+, intltool
+, gettext
+, taglib
+}:
+
+mkXfceDerivation {
+  category = "thunar-plugins";
+  pname = "thunar-media-tags-plugin";
+  version = "0.3.0";
+
+  sha256 = "sha256-jtgcHH5U5GOvzDVUwPEreMtTdk5DT6sXvFPDbzbF684=";
+
+  nativeBuildInputs = [
+    intltool
+    gettext
+  ];
+
+  buildInputs = [
+    thunar
+    exo
+    gtk3
+    libxfce4util
+    taglib
+  ];
+
+  preConfigure = ''
+    ./autogen.sh
+  '';
```

Comment:
This could be replaced with autoreconfHook.

Reply:
Right, it seems to just work if I remove the `preConfigure` hook, do you know why? Should I still include `autoreconfHook` explicitly?

Reply:
Ok so `mkXfceDerivation` pulls `xfce4-dev-tools` which pulls `autoreconfHook`.